rageval/metrics/_answer_rouge.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+
+from typing import List, Any, Callable, Union
+from rouge_score import rouge_scorer
+
+from datasets import Dataset
+from dataclasses import dataclass
+
+from rageval.metrics import Metric, add_attribute
+
+
+@dataclass
+@add_attribute('mtype', 'AnswerCorrectness')
```

Comment:
The latest PR has reformatted the docstring of class using datasets.utils.file_utils.add_start_docstring.